src/main/java/net/mcreator/ui/browser/WorkspaceFileBrowser.java

```diff
@@ -330,7 +284,75 @@ public void reloadTree() {
 		}
 	}
 
-	private void loadExtSoruces(FilterTreeNode node) {
+	public void openSelectedFile(boolean forceExpansion) {

```

Comment:
Some documentation on arguments like forceExpansion wouldn't hurt, as the name itself does not really explain well the function of the argument

Reply:
Then I had better attempt to document all class, not just this method.

Reply:
I documented all public methods I found in the class. Do you think it looks good?